src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -52,6 +52,7 @@ export interface ReviewProps extends SystemContextProps {
   dialog: Dialog
   commitMutation: CommitMutation
   isCommittingMutation: boolean
+  user?: User
```

Comment:
you don't need this,`SystemContextProps` already provides it and it's not possible to be undefined from there

Reply:
Good to know! Will remove this in the incoming PR.